tests/test_rxerr.cpp

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (c) 2016-2019 UAVCAN Team
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * Contributors: https://github.com/UAVCAN/libcanard/contributors
+ */
+
+#include <catch.hpp>
+#include <canard.h>
+
+//Independent implementation of ID layout from https://uavcan.org/Specification/4._CAN_bus_transport_layer/
+#define CONSTRUCT_PRIO(prio)        (((prio) & 0x1F) << 23)
```

Comment:
This need to be bit-shifted with 24 to match the v0.1 specification.

Reply:
This is actually not picked up due to none of the tests using any prio other than `0`. This should probably be both fixed and added a test case for in master. Then backported into v1.0 as well.

Reply:
@pavel-kirienko there is a fix in #93 

I think if you merge it + include that change as well we're ready to land this PR

Reply:
Thanks, done.